### PR TITLE
some corrections to test

### PR DESCRIPTION
**Pull request by ake123**
There is an error in `devtools::test()` where some functions were not updated on the unit test in `plot series`
Done some corrections `plot series `test
related to issue `Switch to mia data (remove microbiomeDataSets data) #141`  #141

### CODE REVIEW

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
Any updates on this? Checks are failing.

**Comment by ake123** [comment]
It seems to be working but fails for ubuntu deployment.

File: tests/testthat/test-2plotSeries.R [hunk 1]
```diff
@@ -1,104 +1,126 @@
...
     tse <- SilvermanAGutData
-    tse_sub <- tse[1:5]
+    tse_sub <- tse[1:5, ]
     
     # Expect error
     expect_error(plotSeries())
     expect_error(plotSeries(tse_sub))
     
-    # Expect output
-    expect_type(plotSeries(tse_sub, assay.type = "counts", x = "DAY_ORDER"), "list")
```
Comment (TuomasBorman) [inline tests/testthat/test-2plotSeries.R:15]: Why the output type was changed? And when?

**Review by TuomasBorman — APPROVED** [review]
Seems good

**Pull request merged**